java/rocksjni/rocksjni.cc

```diff
@@ -1289,6 +1290,52 @@ void Java_org_rocksdb_RocksDB_deleteRange__J_3BII_3BII(
                               jend_key, jend_key_off, jend_key_len);
 }
 
+
+/*
+ * Class:     org_rocksdb_RocksDB
+ * Method:    deleteFilesInRange
+ * Signature: (J[BII[BII)V
+ */
+JNIEXPORT void JNICALL Java_org_rocksdb_RocksDB_deleteFilesInRange(
```

Comment:
`JNIEXPORT` and `JNICALL` need to be removed from here

Reply:
👍  will do